script/make-pretty

```diff
@@ -143,10 +143,10 @@ do_shell_check()
     echo -e '     check shell'
     echo -e '====================='
 
-    shfmt -f . | grep -v -E "^($(echo "${OT_EXCLUDE_DIRS[@]}" | tr ' ' '|'))" \
+    shfmt -f . | grep -v -E "^($(echo "${OT_EXCLUDE_PATTERNS[@]}" | tr ' ' '|'))" \
```

Comment:
I think using git ls-files is better which avoid changing untracked files.

Reply:
Agree.